CHANGELOG.rst

```diff
@@ -6,6 +6,14 @@ theforeman.foreman Release Notes
 
 This changelog describes changes after version 0.8.1.
 
+v3.13.0
+=======
+
+Minor Changes
+-------------
+
+- Add a wait for task module
```

Comment:
This (and the change to `changelogs/changelog.yaml`) are not necessary, they will be automatically generated when we do the release.

Reply:
Removed